rtengine/procparams.h

```diff
@@ -2396,6 +2401,14 @@ class ProcParams
 
     bool operator ==(const ProcParams& other) const;
     bool operator !=(const ProcParams& other) const;
+    template <class T1, class T2, size_t N> int posInArray(const std::array<T1,N>& arr, const T2& val, int defaultValue = 0) const {
```

Comment:
This is generic enough to put it somewhere else.

Reply:
@Floessie Do you have a suggestion where to put it best?